drizzle-orm/src/sqlite-core/README.md

```diff
@@ -19,7 +19,8 @@ Here you can find extensive docs for SQLite module.
 | [bun:sqlite](https://github.com/oven-sh/bun#bunsqlite-sqlite3-module) | ✅ | [Example](https://github.com/drizzle-team/drizzle-orm/tree/main/examples/bun-sqlite)| |
 | [Cloudflare D1](https://developers.cloudflare.com/d1/) | ✅ | [Example](https://github.com/drizzle-team/drizzle-orm/tree/main/examples/cloudflare-d1)| |
 | [Fly.io LiteFS](https://fly.io/docs/litefs/getting-started/) | ✅ | | |
-| [ChiselStrike Turso](https://chiselstrike.com/) | ✅ | [Example](https://github.com/drizzle-team/drizzle-orm/tree/main/examples/cloudflare-d1) | |
+| [libSQL server](https://github.com/libsql/sqld/) | ✅ | [Example](../../../examples/libsql) | |
+| [Turso](https://turso.tech/) | ✅ | [Example](../../../examples/libsql) | |
```

Comment:
These can be absolute paths - `/examples/libsql`

Reply:
Thanks! I force-pushed a change with that and your other comment